src/plot_utils.py

```diff
@@ -321,7 +318,7 @@ def guess_column_type(df_column):
         return guess_column_type(df_column)
 
 
-def conv_categorical_vals_to_numeric(data_srs, bdb=None, generator_name=None):
+def conv_categorical_vals_to_numeric(data_srs):
     # TODO: get real valuemap from btable
```

Comment:
Date: Tue, 29 Sep 2015 15:35:21 -0700
   From: gregory-marton <notifications@github.com>

   > @@ -321,7 +318,7 @@ def guess_column_type(df_column):
   >          return guess_column_type(df_column)
   >  
   >  
   > -def conv_categorical_vals_to_numeric(data_srs, bdb=None, generator_name=None):
   > +def conv_categorical_vals_to_numeric(data_srs):
   >      # TODO: get real valuemap from btable

   No, I don't. I don't especially want to remove the TODO comment even if it is related, and I think reinstating the arguments if they're relevant would be pretty easy.

The TODO comment has to do with getting the codebook's value map out
of the table in order to display the values of categorical columns
more prettily, e.g. to display `GEO' as `Geosynchronous orbit'.  That
is presumably why the generator was passed in.

Really it should be gotten not by passing in the generator, but via
the cursor's cursor.description -- if bayeslite were made to pass
statistical types and associated metadata (as in the codebook) through
to cursors the way sqlite3 passes declared column types through to
cursors.